test/bot.coffee

```diff
@@ -46,33 +46,33 @@ describe 'Logger', ->
 describe 'Send Messages', ->
 
   it 'Should send a message', ->
-    sentMessages = @slackbot.send {room: 'general'}, 'message'
+    sentMessages = @slackbot.send {room: @stubs.channel.id}, 'message'
     sentMessages.length.should.equal 1
     sentMessages[0].should.equal 'message'
 
   it 'Should send multiple messages', ->
-    sentMessages = @slackbot.send {room: 'general'}, 'one', 'two', 'three'
+    sentMessages = @slackbot.send {room: @stubs.channel.id}, 'one', 'two', 'three'
     sentMessages.length.should.equal 3
 
   it 'Should not send empty messages', ->
-    sentMessages = @slackbot.send {room: 'general'}, 'Hello', '', '', 'world!'
+    sentMessages = @slackbot.send {room: @stubs.channel.id}, 'Hello', '', '', 'world!'
     # Removes undefined (or unsent) messages from sentMessages
     _.remove(sentMessages, _.isUndefined)
     sentMessages.length.should.equal 2
 
   it 'Should not fail for inexistant user', ->
-    chai.expect(() => @slackbot.send {room: 'inexistant'}, 'Hello').to.not.throw()
+    chai.expect(() => @slackbot.send {room: 'U987'}, 'Hello').to.not.throw()
```

Comment:
nitpicky again, but shouldn't this value look like a conversation ID (not a user ID)? or maybe that was what is intended, and then i think a comment could help clear that up. ditto for many of the edits below.

Reply:
The `Client sending message` tests should be conversation ID and I changed it to reflect that, but some of the `Send messages` test are trying to send a DM to a user with no conversation ID. It was using `username` before, so now a few of those are using user ID. That being said, the current tests are limited because of lost functionality (`@client.openDM()`)